plugins/sqlfluff-templater-dbt/sqlfluff_templater_dbt/templater.py

```diff
@@ -97,21 +97,15 @@ def dbt_version(self):  # pragma: no cover
     @cached_property
     def dbt_config(self):
         """Loads the dbt config."""
-        # Here, we read flags.PROFILE_DIR directly, prior to calling
-        # set_from_args(). Apparently, set_from_args() sets PROFILES_DIR
-        # to a lowercase version of the value, and the profile wouldn't be
-        # found if the directory name contained uppercase letters. This fix
-        # was suggested and described here:
-        # https://github.com/sqlfluff/sqlfluff/issues/2253#issuecomment-1018722979
-        user_config = read_user_config(flags.PROFILES_DIR)
         flags.set_from_args(
             DbtConfigArgs(
                 project_dir=self.project_dir,
                 profiles_dir=self.profiles_dir,
                 profile=self._get_profile(),
                 vars=self._get_cli_vars(),
+                threads=self._get_threads(),
             ),
-            user_config,
+            user_config=None,
```

Comment:
For safety (I think), let's keep it as a positional parameter. Otherwise, we may be passing it as a different parameter after this change.
```suggestion
            None,
```